src/main/java/gfhouse/matchmaker/controller/UserController.java

```diff
@@ -0,0 +1,25 @@
+package gfhouse.matchmaker.controller;
+
+import gfhouse.matchmaker.domain.User;
+import gfhouse.matchmaker.dto.UserDto;
+import gfhouse.matchmaker.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+
+@RestController
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("/signup")
+    public User signup(@RequestBody UserDto userDto){
```

Comment:
```suggestion
    public User signup(@RequestBody UserDto userDto) {
```
일관된 코드 스타일을 유지하는 것이 좋겠습니다.